src/NServiceBus.Core/Performance/Statistics/ProcessingStatisticsBehavior.cs

```diff
@@ -29,7 +29,7 @@ class ProcessingStatisticsBehavior : IBehavior<IIncomingPhysicalMessageContext,
             finally
             {
                 stopwatch.Stop();
-                state.ProcessingEnded = state.ProcessingStarted.AddTicks(stopwatch.ElapsedTicks);
+                state.ProcessingEnded = state.ProcessingStarted.AddTicks(stopwatch.Elapsed.Ticks);
```

Comment:
Looking at how `AddTicks` and the DateTime `operator +` overload are implemented, they work out to being the same thing.

@ramonsmits suggestion is a bit clearer overall though 👍